frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -131,8 +137,24 @@ const AdminShoutouts: React.FC = () => {
     if (view === 'PRESENT')
       return (
         <Item.Group divided>
+          <header className={styles.presentCount}>
+            <img
```

Comment:
It is recommended that you use the `Image` component provided by SemanticUI: https://react.semantic-ui.com/elements/image/.